src/main/java/com/zebrunner/carina/demo/mobile/gui/pages/android/ProductListPage.java

```diff
@@ -37,6 +48,35 @@ public boolean isNumberOfItemsInCartCorrect(Integer enteredNumber){
         return enteredNumber.equals(Integer.parseInt(cartItemCount.getText()));
     }
 
+    private ArrayList<Float> getPrices() {
```

Comment:
On ProductListPage you should have ProductListItemComponent, and inside this component you can getPrices and all elements related to this component